.github/workflows/pr-check.yml

```diff
@@ -111,10 +111,10 @@ jobs:
         yarn run http-server dist/ -p 8080 -s &
         sleep 5
         curl http://localhost:8080
-        yarn run testcafe -c 1 "browserstack:chrome:Windows 10" testcafe/*.spec.ts
-        yarn run testcafe -c 1 "browserstack:firefox:Windows 10" testcafe/*.spec.ts
-        yarn run testcafe -c 1 "browserstack:safari:OS X Catalina" testcafe/*.spec.ts --skip-js-errors
-        yarn run testcafe -c 1 "browserstack:Samsung Galaxy S8" testcafe/*.spec.ts
+        yarn run testcafe -c 1 "browserstack:chrome:Windows 10" testcafe/*.spec.ts testcafe/*.tests.ts
```

Comment:
I would try to be consistent with the file names and either always use .spec.ts or always .test.ts. I would vote for spec.ts.

Reply:
I would consider a third option `*.e2e.ts`

Reply:
I've preferred .test.ts because it have been choose by DevExpress and sounds like tests files.
I've never seen the "spec" extention before what does it choosed for?
For sure one convention ist better, I let you choose @ChristophWersal.

Reply:
File has been renamed, feel free to change the name if you prefer another convention.

Reply:
Thanks. 
*.jest.ts is the standard used by many projects that work with jest, like create-react-app. I assume that spec stands for specification, since the tests should specify the behaviour of the code.